cookbook-efforts/multilingual-prompt-engine/README.md

```diff
@@ -0,0 +1,131 @@
+
+These responses are then:
+
+1. Used to create novel datasets which contain both prompts and responses.
+2. Uploaded to an [Argilla Space](https://huggingface.co/docs/hub/en/spaces-sdks-docker-argilla) for ranking by the community.
+
+The latter effort will enable the creation of novel, language-specific performance metrics for multilingual LLMs and therefore directly contribute to the advancement of multilingual AI.
+
+## Why do we need multilingual metrics and why does multilingual performance matter?
+
+[Good AI requires good data](https://huggingface.co/blog/ethics-soc-6). Building meaningful multilingual benchmarks contributes to the creation of good datasets upon which foundation models may be trainied to be performative in languages other than English.
+
+Further to that point, the [well-known English-language bias](https://www.axios.com/2023/09/08/ai-language-gap-chatgpt) in AI means that [traditional metrics](https://huggingface.co/docs/leaderboards/open_llm_leaderboard/about) do not address model performance in handling prompts in languages other than English.
+
+The MPEP initiative is taking two meaningful steps to address these issues:
+
+1. Creating training data for models to handle prompts in languages other than English.
+2. Creating a mechanism by which existing models may be scored based on their performance in specific languages other than English.
+
+## How can you contribute?
+
+Community members can contribute to the MPEP initiative (and MPE project) in a number of ways:
+
+1. Contribute to the MPE codebase: Just open a PR with your changes and a short explanation of what you have done.
+2. Contribute to the MPEP initiative as an annotator or language lead: Check out the [MPEP README](../../community-efforts/prompt_translation/README.md) for more information about which languages are already in progress, if you don't see your language, become a language lead, gather a group of annotators and build the dataset!
+3. Contribute to the annotation of MPE results: The end product of the MPE project is an Argilla Space with each prompt and response across all languages and models. The annotating user identifies the language in which they would like to rank responses, and then offers a 1-5 ranking of the quality of the response to include judgements on quality of language use, syntax, vocabulary, grammar, and thoroughness of the response itself.
+
+## Project Overview
+
+This section of the documentation offers a configuration example of [`01_create_response_dataset.ipynb`](./01_create_response_dataset.ipynb) so that you may modify the code locally to prompt models of your choosing with datasets of your choosing as well.
```

Comment:
```suggestion
This section of the documentation offers a configuration example of [`01_create_response_dataset.ipynb`](./01_create_response_dataset.ipynb) so that you may modify the code locally to prompt models of your choosing with datasets of your choosing as well.
```